pages/common/html5validator.md

```diff
@@ -0,0 +1,32 @@
+# html5validator
+
+> A command line tool for testing HTML5 validity.
+> More information: <https://github.com/svenkreiss/html5validator>.
+
+- Validate a specific file:
+
+`html5validator {{path/to/file}}`
+
+- Validate all HTML files in a specific directory:
+
+`html5validator --root {{path/to/directory}}`
+
+- Show warnings as well as errors:
```

Comment:
I would say we should have stated that it will "Validate showing warnings and errors" but after two examples I think that's implicit.

Reply:
Yep